ocs_ci/deployment/deployment.py

```diff
@@ -1127,6 +1127,9 @@ def setup_local_storage(storageclass):
     storage_class_device_count = 1
     if platform == constants.AWS_PLATFORM:
         storage_class_device_count = 2
+    if platform == constants.IBM_POWER_PLATFORM:
+        numberofstoragedisks = config.ENV_DATA['number_of_storage_disks']
```

Comment:
```suggestion
        numberofstoragedisks = config.ENV_DATA.get("number_of_storage_disks", 1)
```

We don't want a `KeyError` if someone doesn't use `production_powervs_upi.yaml`. This also means we can drop the value from that file as well.

Reply:
The motive behind doing this was , we can update our config file in our case its (production_powervs_upi.yaml) according to the type of cluster we are using. We are using both libvirt and powervs clusters , and in both case number_of_storage_disks varies .  ie. in libvirt cluster no. of storage disks is 1, whereas for powervs its 8. By default i set it to 1 , so that it will not effect our ocs-upi-kvm project . 
If we don't use number_of_storage_disks in production_powervs_upi.yaml then we have to update deployment.py each time depending on the type of cluster we use . So i think good approach is updating config file rather than updating python file(deployment.py)  . Do you still want me to make the changes that you suggested, if yes then please let me know.
And while running run-ci for power , we are specifying production_powervs_upi.yaml as our config file. 
run-ci command we are using is : 

`run-ci -m "tier4a and manage" --ocsci-conf=conf/ocsci/production_powervs_upi.yaml --cluster-name "ocstest" --cluster-path /root tests/`